### PR TITLE
set gyp.devDir so headers are downloaded correctly

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/mafintosh/prebuild/issues/172

### CODE REVIEW

File: test/gypbuild-test.js [hunk 1]
```diff
@@ -43,6 +44,8 @@ test('gyp is invoked with correct arguments, release mode', function (t) {
     log: noop
   }
   runGyp(opts, 'x.y.z', function (err) {
+    var devDir = opts.gyp.devDir
+    t.equal(devDir.indexOf(osenv.home()), 0, devDir + ' should start with home folder')
```
Comment (ralphtheninja) [inline test/gypbuild-test.js]: It's a little bit silly sometimes when you write test code that's basically the same as the code you wrote.

**Comment by ralphtheninja** [comment]
cc @mathiask88

**Comment by mathiask88** [comment]
Maybe we should give the user the opportunity to pass a `--devdir` argument accordingly to `node-gyp`. Otherwise LGTM.

**Comment by ralphtheninja** [comment]
I hear you, at the same time I want to add as little complexity as possible until someone really needs to configure `~/.node-gyp` to something else.

**Comment by ralphtheninja** [comment]
6.2.1

**Pull request merged**